base/deprecated.jl

```diff
@@ -144,6 +144,8 @@ end
 @deprecate chol(A::Number, ::Type{Val{:L}})         ctranspose(chol(A))
 @deprecate chol(A::AbstractMatrix, ::Type{Val{:L}}) ctranspose(chol(A))
 
+@deprecate cumop! scan!
```

Comment:
Right, is the policy that only exported functions get deprecated? Should I remove this line?